formatter/formatter.go

```diff
@@ -65,8 +69,17 @@ func JUnitReportXML(report *parser.Report, noXMLHeader bool, goVersion string, w
 
 	// convert Report to JUnit test suites
 	for _, pkg := range report.Packages {
+		var tests int
+		if len(pkg.Tests) >= 1 && len(pkg.Benchmarks) >= 1 {
```

Comment:
I think this entire if-statement can be simplified to just 
```go
len(pkg.Tests) + len(pkg.Benchmarks)
```
This can be done inline when creating the `JUnitTestSuite` struct below.